### PR TITLE
Fix #114: Switch from SCSS to CSS

### LINKED ISSUE #114: Convert SCSS to CSS
The main things to address/make sure they don't break are:

- General colors, spacing, styles
- Variables
- Nesting
- Uninstalling `sass` from package.json

### PR DESCRIPTION
**Pull request by palashkaria**
fixes #114 

- uses CSS variables (imported separately in `main.jsx`)
- renamed all files to `.css`, switch variable usage and remove `variable.scss` import 
- removes sass from package.json
- uses postcss-nested for nesting

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,7 +67,8 @@
 		"eslint": "^8.26.0",
 		"eslint-plugin-react": "^7.31.10",
 		"jsdom": "^20.0.1",
-		"sass": "^1.55.0",
+		"postcss": "^8.4.21",
+		"postcss-nested": "^6.0.0",
```
Comment (cassidoo) [inline package.json:71]: Didn't know this package existed, awesome!

**Review by cassidoo — COMMENTED** [review]
Looks good to me at first glance, thank you!! I'm going to run it on a couple different operating systems before shipping it, but it looks great.

If I make a change it might only be to remove the nesting. Weighing a bit of the pros and cons of installing a package to keep the nesting or not.

**Comment by palashkaria** [comment]
> Looks good to me at first glance, thank you!! I'm going to run it on a couple different operating systems before shipping it, but it looks great.
> 
> If I make a change it might only be to remove the nesting. Weighing a bit of the pros and cons of installing a package to keep the nesting or not.

I'd think nesting works well, and since [`postcss-nested`](https://github.com/postcss/postcss-nested) is only build-time, there's no real cost to it. I'd keep it for readability? But could be removed if you think that's better

**Comment by cassidoo** [comment]
Thank you again!

**Pull request merged**